e2e_tests/standard/test_base.py

```diff
@@ -199,36 +195,9 @@ def test_fetch_runs_table(self, environment):
             key=lambda r: r.get_attribute_value("sys/id"),
         )
         assert len(runs_table) == 1
```

Comment:
Would be nice to check at least id.

Reply:
Why? `_id` is some internal detail, isn't correct `sys/id` enough?

Reply:
This is what I meant. But right now only number of runs is asserted.